docs/rte-mac-setup.md

```diff
@@ -1,10 +1,16 @@
 # Setup MAC adress at U-Boot
 
 After flashing the card with the new image, the `MAC` will be generated from the
-board's serial id and stored in `U-Boot`'s environment variable on the first
+board's `Serial ID` and stored in `U-Boot`'s environment variable on the first
 boot.
 
-You can change the address for a single boot by modifying the variable from
+If no `Serial ID` is found in U-Boot environment, tha `MAC` will be regenerated.
+
+If `Serial ID` change, the `MAC` will be regenerated again.
```

Comment:
@leckijakub please explain here how the Serial ID can be changed (same SD card, different device)

Reply:
@TomaszAIR explanation added.